packages/flutter/test/rendering/wrap_test.dart

```diff
@@ -12,17 +12,146 @@ void main() {
+          '   alignment: start\n'
+          '   spacing: 0.0\n'
+          '   runAlignment: start\n'
+          '   runSpacing: 0.0\n'
+          '   crossAxisAlignment: 0.0\n'),
     );
   });
+
+  test(
+    'Compute intrinsic height test',
+    () {
+      final List<RenderBox> children = <RenderBox>[
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+      ];
+
+      _intrinsicHeightDefaultTest(children, 245, 165);
+      _intrinsicHeightDefaultTest(children, 250, 80);
+      _intrinsicHeightDefaultTest(children, 80, 250);
+      _intrinsicHeightDefaultTest(children, 79, 250);
+      _intrinsicHeightDefaultTest(children, 245, 165, isMax: false);
+      _intrinsicHeightDefaultTest(children, 250, 80, isMax: false);
+      _intrinsicHeightDefaultTest(children, 80, 250, isMax: false);
+      _intrinsicHeightDefaultTest(children, 79, 250, isMax: false);
+    },
+  );
+
+  test(
+    'Compute intrinsic width test',
+    () {
+      final List<RenderBox> children = <RenderBox>[
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
+        ),
+      ];
+
+      _intrinsicWidthDefaultTest(children, 245, 165);
+      _intrinsicWidthDefaultTest(children, 250, 80);
+      _intrinsicWidthDefaultTest(children, 80, 250);
+      _intrinsicWidthDefaultTest(children, 79, 250);
+      _intrinsicWidthDefaultTest(children, 245, 165, isMax: false);
+      _intrinsicWidthDefaultTest(children, 250, 80, isMax: false);
+      _intrinsicWidthDefaultTest(children, 80, 250, isMax: false);
+      _intrinsicWidthDefaultTest(children, 79, 250, isMax: false);
+    },
+  );
+}
+
+void _intrinsicWidthDefaultTest(
+  List<RenderBox> children,
+  double availableHeight,
+  double expectedWidth, {
+  Axis direction = Axis.vertical,
+  double spacing = 5.0,
+  double runSpacing = 5.0,
+  bool isMax = true,
+}) {
+  final RenderWrap renderWrap = _prepareWrapForComputeIntrinsicSize(
+    children,
+    spacing: spacing,
+    runSpacing: runSpacing,
+    direction: direction,
+  );
+  expect(
+    isMax
+        ? renderWrap.computeMaxIntrinsicWidth(availableHeight)
+        : renderWrap.computeMinIntrinsicWidth(availableHeight),
+    equals(expectedWidth),
+  );
+
+  // clear for available reuse children in test
```

Comment:
Not sure what you mean by this comment? It should be a full sentence (starting with a capital letter and ending with a `.`), though.